pkg/api/constant.go

```diff
@@ -1,6 +1,12 @@
 package api
 
 const (
+	ClusterNameAPICI   = "api.ci"
```

Comment:
Why do we have these _and_ the typed ones?

Reply:
Removed the string ones.